### PR TITLE
Prevents holstering weapons while attack delayer is blocked, also holster verb name quality

### PR DESCRIPTION
**Pull request by 9600bauds**
Some people would abuse holsters by unholster->click->instaholster so they wouldn't lose their gun if stunned, you couldn't do this with containers before so whoop whoop
Arguably more of a bugfix than a balance change but you decide

### CODE REVIEW

**Comment by Exxion** [comment]
Why not both?

File: code/modules/clothing/accessories/holster.dm [hunk 1]
```diff
@@ -51,15 +54,16 @@
 		return
 
 	var/obj/item/clothing/accessory/holster/H = null
-	if (istype(src, /obj/item/clothing/accessory/holster))
+	if(istype(src, /obj/item/clothing/accessory/holster))
```
Comment (ComicIronic) [inline code/modules/clothing/accessories/holster.dm:58]: Nice non-standard formatting changes

Comment (9600bauds) [inline code/modules/clothing/accessories/holster.dm:58]: Thank you

**Comment by clusterfack** [comment]
Cheating macroing bastards

**Pull request merged**